packages/esm-patient-chart-app/src/clinical-views/utils/index.ts

```diff
@@ -35,7 +35,7 @@ export const getEncounterTileColumns = (tileDefinition: MenuCardProps, config: C
           config: config,
         });
       }
-      return typeof obsValue === 'string' ? obsValue : obsValue?.name?.name ?? '--';
+      return typeof obsValue === 'string' || typeof obsValue === 'number' ? obsValue : obsValue?.name?.name ?? '--';
```

Comment:
```suggestion
      return typeof obsValue === 'string' || (typeof obsValue === 'number' && !isNaN(obsValue)) 
        ? obsValue 
        : obsValue?.name?.name ?? '--';
```

Would this edit work a touch better?